gensim/similarities/docsim.py

```diff
@@ -74,8 +74,7 @@
 
 
 class Shard(utils.SaveLoad):
-    """
-    A proxy class that represents a single shard instance within a Similarity
+    """A proxy class that represents a single shard instance within a Similarity
```

Comment:
120 character limit (not 80)
Also, for definitions, first line of docstring **must** end with `.`